### PR TITLE
ZCS-9704: Status of the response from Location and Equipment is not shown in tooltip

### PR DESCRIPTION
**Pull request by madhavdhuppe**

### CODE REVIEW

File: WebRoot/js/zimbraMail/calendar/model/ZmAppt.js [hunk 1]
```diff
@@ -277,40 +313,36 @@ function(controller, callback) {
...
-			params.attendeesText = this.getAttendeeToolTipString(attendees);
-		}
+	if (this.equipmentPtStHashMap != null) {
+		var equipmentAttributes = this._getItemAttributes(this.equipmentPtStHashMap, ZmCalBaseItem.EQUIPMENT);
+		params.hideEquipments = equipmentAttributes.hideAttendees;
+		params.equipmentPtStStatus = equipmentAttributes.ptstStatus;
+		params.equipmentText = equipmentAttributes.attendeesText;
 	}
 
+	console.log(this, personAttributes, locationAttributes, equipmentAttributes);
```
Comment (auminpatel) [inline WebRoot/js/zimbraMail/calendar/model/ZmAppt.js:344]: please remove the console

Comment (madhavdhuppe) [reply WebRoot/js/zimbraMail/calendar/model/ZmAppt.js:344]: ohh removed

**Review by auminpatel — APPROVED** [review]
LGTM

**Review by sachinpatel88 — APPROVED** [review]

**Review by BhavinBathani — APPROVED** [review]

File: WebRoot/js/zimbraMail/calendar/model/ZmAppt.js [hunk 2]
```diff
@@ -252,6 +252,42 @@ function(controller, callback) {
...
 
+ZmAppt.prototype._getItemAttributes =
+function (hashMap, itemType) {
+		var ptstStatus = {};
+		var statusAttendees;
+		var attendeesText;
+		var hideAttendees = true;
+
+		statusAttendees = ptstStatus[ZmMsg.ptstAccept] = this._getPtstStatus(hashMap, ZmCalBaseItem.PSTATUS_ACCEPT);
+		hideAttendees = hideAttendees && !statusAttendees.count;
```
Comment (silentsakky) [inline WebRoot/js/zimbraMail/calendar/model/ZmAppt.js:263]: do we need to define statusAttendees ? we can use ptstStatus[ZmMsg.ptstAccept].count instead

File: WebRoot/templates/calendar/Appointment.template [hunk 3]
```diff
@@ -995,16 +996,94 @@
...
+						<$= AjxStringUtil.htmlEncode(attendeesPtStStatus[i].attendees) $>
+					</td>
+				</tr>
+		<$
+				}
+			}
+	}
+	$>
+
+	<$ if (!data.appt.isDraft && data.appt.isOrganizer() && data.appt.otherAttendees && !data.hideLocations) { $>
```
Comment (silentsakky) [inline WebRoot/templates/calendar/Appointment.template:1017]: we are duplicating same block of code, please check to reduce duplication

Comment (madhavdhuppe) [reply WebRoot/templates/calendar/Appointment.template:1017]: reduced code

File: WebRoot/js/zimbraMail/calendar/model/ZmAppt.js (hunk 2 above)
Comment (madhavdhuppe) [reply WebRoot/js/zimbraMail/calendar/model/ZmAppt.js:263]: removed

**Review by silentsakky — APPROVED** [review]

**Pull request merged**